core/jobs/batch_jobs/suggestion_edit_state_content_deletion_jobs_test.py

```diff
@@ -0,0 +1,284 @@
+        self.suggestion_3_model.update_timestamps()
+        self.suggestion_4_model.update_timestamps()
+        suggestion_models.GeneralSuggestionModel.put_multi([
+            self.suggestion_1_model,
+            self.suggestion_2_model,
+            self.suggestion_3_model,
+            self.suggestion_4_model])
+
+        queries = [(
+            'suggestion_type',
+            feconf.SUGGESTION_TYPE_EDIT_STATE_CONTENT)]
+
+        self.assertEqual(
+            len(suggestion_models.GeneralSuggestionModel.query_suggestions(
+                queries)), 2)
+
+        self.assert_job_output_is([
+            job_run_result.JobRunResult(
+                stdout='EDIT STATE CONTENT SUGGESTION SUCCESS: 2')
+        ])
+
+        self.assertEqual(
+            len(suggestion_models.GeneralSuggestionModel.query_suggestions(
+                queries)), 0)
+
+
+class AuditDeleteDeprecatedSuggestionEditStateContentModelsJobTests(
+    job_test_utils.JobTestBase):
+
+    jobs = suggestion_edit_state_content_deletion_jobs
+
+    JOB_CLASS: Type[
+        jobs.AuditDeprecatedSuggestionEditStateContentModelsDeletionJob
+    ] = jobs.AuditDeprecatedSuggestionEditStateContentModelsDeletionJob
+
+    def create_suggestion(
+            self,
+            suggestion_type: str,
+            target_type: str,
+            target_id: str,
+            author_id: str,
+            final_reviewer_id: str,
+            language_code: None | str
+            ) -> suggestion_models.GeneralSuggestionModel:
+        """creates new suggestion.
+
+        Args:
+            suggestion_type: str. Type of the suggestion that is being
+                created.
+            target_type: str. Type of traget entity for which the suggestion
+                is being created.
+            target_id: str. The ID of the target entity.
+            author_id: str. The ID of user who creates the suggestion.
+            final_reviewer_id: str. The user ID of the reviewer.
+            language_code: None | str. Code of the language in which the
+                suggestion is created.
+
+        Returns:
+            GeneralSuggestionModel. Returns newly created
+            GeneralSuggestionModel.
+            """
+
+        return self.create_model(
+            suggestion_models.GeneralSuggestionModel,
+            suggestion_type=suggestion_type,
+            target_type=target_type,
+            target_id=target_id,
+            target_version_at_submission=1,
+            status='accepted',
+            author_id=author_id,
+            final_reviewer_id=final_reviewer_id,
+            change_cmd={},
+            score_category=(
+                suggestion_models.SCORE_TYPE_TRANSLATION +
+                suggestion_models.SCORE_CATEGORY_DELIMITER + 'English'),
+            language_code=language_code
+        )
+
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.suggestion_1_model: suggestion_models.GeneralSuggestionModel = (
```

Comment:
Instead of numbering the variables, is it possible to make the names provide more context about what it represents? e.g. state content suggestions & question suggestion.

Reply:
done